src/repository/task.repository.ts

```diff
@@ -0,0 +1,44 @@
+import Task from "../models/Task";
+import Repository from "../interfaces/repository.interface";
+import { v4 as uuidv4 } from "uuid";
+
+let tasks: Task[] = [];
+
+class TaskRepository implements Repository {
```

Comment:
Du coup tu peux utiliser l'interface générique :

```suggestion
class TaskRepository implements Repository<Task> {
```